src/test/ts/alien/TestHelper.ts

```diff
@@ -0,0 +1,27 @@
+import { Global, Strings, Arr } from '@ephox/katamari';
+import { SugarElement, Attribute, SelectorFilter, Remove } from '@ephox/sugar';
+import { ScriptLoader } from '../../../main/ts/ScriptLoader';
+
+const VALID_API_KEY = 'qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc';
+
+// Function to clean up and remove TinyMCE-related scripts and links from the document
+const cleanupGlobalTinymce = () => {
+  ScriptLoader.reinitialize();
+  // Delete global references to TinyMCE, if they exist
```

Comment:
Just a nitpick but I'm not sure how necessary this comment is since it can be inferred on the next two lines.

Maybe a comment explaining why we remove these references would be more helpful?

Reply:
```suggestion
  // This deletes global references to TinyMCE, to ensure a clean slate for each initialization when tests are switching to a different editor version.
```